docs/config.md

```diff
@@ -647,6 +647,19 @@ of the target PostgreSQL cluster.
       INHERIT: yes
 ```
 
+#### `config`  { #role-config }
```

Comment:
Please add a new line between title and text.

Reply:
Done